test/Microsoft.ML.Benchmarks/Text/MultiClassClassification.cs

```diff
@@ -130,4 +95,47 @@ public void CV_Multiclass_WikiDetox_WordEmbeddings_SDCAMC()
             }
         }
     }
+
+    [Config(typeof(PredictConfig))]
+    public class MultiClassClassificationTest
+    {
+        private string _dataPath_Wiki;
+        private string _modelPath_Wiki;
+
+        [GlobalSetup]
+        public void SetupScoringSpeedTests()
+        {
+            _dataPath_Wiki = Path.GetFullPath(TestDatasets.WikiDetox.trainFilename);
+
+            if (!File.Exists(_dataPath_Wiki))
+                throw new FileNotFoundException(string.Format(Helpers.DatasetNotFound, _dataPath_Wiki));
+
+            _modelPath_Wiki = Path.Combine(Directory.GetCurrentDirectory(), @"WikiModel.zip");
```

Comment:
Do we want to cleanup these artifacts after the tests complete, or is leaving the intermediate files of tests in place the convention? Currently, each time this runs, the WikiModel.zip is overwritten and kept.

This is more of a question of: what is the convention, leaving files in place, or cleaning up test folders?

Reply:
in dotnet we generally clean up the files. I can delete the model file in GlobalCleanup function